Goobi/plugins/opac/PicaPlugin/org/goobi/production/plugin/CataloguePlugin/PicaPlugin/PicaPlugin.java

```diff
@@ -234,19 +236,19 @@ public static String getDescription(Locale language) {
 		Type type;
 		Fileformat ff;
 		try {
-			/*
-			 * -------------------------------- Opac abfragen und erhaltenes
-			 * Dom-Dokument in JDom-Dokument umwandeln
+			/* 
+			 * --------------------------------
```

Comment:
General hint / question for whole pull request: On cleaning up and translations comments, please remove this unnecessary `---`,  `===` or `###` lines. One line comments should started with `//`, two or more line comments should / could be surrounded in `/*` ... `*/`